core/controllers/profile_test.py

```diff
@@ -534,6 +536,31 @@ def test_created(self):
             response_dict['edited_exp_summary_dicts'][0]['id'],
             self.EXP_ID_1)
 
+
+    def test_created_collection(self):
+        # check that the profile page for a user who has created
+        # a single collection shows 2 created and 2 edited.
```

Comment:
Since this comment applies to the entire function, it would be better if this comment were part of the doc string of the test method, see https://github.com/oppia/oppia/blob/develop/core/controllers/profile_test.py#L273